kinto/views/collection.py

```diff
@@ -0,0 +1,10 @@
+from cliquet.resource import crud, BaseResource
+
+
+@crud(path="/collections/{collection_id}/records/{id}",
+      collection_path="/collections/{collection_id}/records")
+class Collection(BaseResource):
+
+    @property
+    def name(self):
+        return self.request.matchdict['collection_id']
```

Comment:
Yeah, cliquet rocks!

We will need a bit more code to handle collections management, plus currently it seems I can only add empty items to the collections (not very handy ;))